ansible-ipi-install/roles/installer/templates/install-config-virtualmedia.j2

```diff
@@ -9,15 +9,30 @@ proxy:
 metadata:
   name: {{ cluster }}
 networking:
-  machineCIDR: {{ extcidrnet }}
+{% if not dualstack_baremetal|bool %}
+  machineCIDR: {{ extcidrnet[0] }}
+{% endif %}
   networkType: {{ network_type }}
-{% if ipv6_enabled|bool and not ipv4_baremetal|bool %}
+{% if ipv6_enabled|bool and not (ipv4_baremetal|bool or dualstack_baremetal|bool) %}
   clusterNetwork:
   - cidr: fd01::/48
     hostPrefix: 64
   serviceNetwork:
   - fd02::/112
 {% endif %}
+{% if ipv6_enabled|bool and dualstack_baremetal|bool %}
+  machineNetwork:
+  - cidr: {{ extcidrnet[0] }}
+  - cidr: {{ extcidrnet[1] }}
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  - cidr: fd02::/48
+    hostPrefix: 64
+  serviceNetwork:
+  - 172.30.0.0/16
+  - fd03::/112
```

Comment:
@rlopez133 @DirectedSoul1 what do you think about hardcoding those values?

Reply:
@iranzo @DirectedSoul1 @smalleni - I'm not a fan of hardcoding values. Makes it harder to fix or find later why something is set. 

As for goal/intention: We want to make sure previous methods of deployments don't change under users. If a FIX or NEW FEATURE is going to change the behavior of deployment and there is no other way, then I suggest making a warning and giving users notice that something is changing in the near future. Likely giving them sometime weeks? a month? to digest the change coming. 

Ideally finding a method that satisfies and doesn't break any previous deployments & gets new method to work as well is what we want to achieve.

Reply:
I agree hardcoding could be harder for future use cases

Reply:
@rlopez133 Thanks for the feedback. I'm not a fan of hardcoding as well, I can always make more variables :) However, I'm wondering if people will even care to change these values. For IPv4, the /23 hostprefix gives 500+ pods per node which is well more than our published limit of 250 pods and our goal of 500 pods per node:) Also, is there a reason why the previous ipv6 serivce ip etc were hardcoded?
```  
clusterNetwork:
  - cidr: fd01::/48
    hostPrefix: 64
  serviceNetwork:
  - fd02::/112
```

Reply:
@DirectedSoul1 @iranzo ^. I am good with taking a different approach to extcidrnet (as stated in previous comments) so it doesn't break existing users, but I  believe we should not variabilize everything. I don't really see value in vairiabilizing these clusternetwork CIDRs, except it adds more variables to the already long inventory.

Reply:
Yes, Service & Cluster network need not be a variable but `machineCIDR`. Lets Create a new var `extcidrnetv6` use it for both `ipv6_enabled` and `dualstack_baremetal` deployment.